src/test/java/com/ksm/wstbackoffice/endpoint/EndPoint.java

```diff
@@ -0,0 +1,9 @@
+package com.ksm.wstbackoffice.endpoint;
+
+public final class EndPoint {
+    public static final String ADDRESSES = "/addresses";
```

Comment:
This is good approach to put constants into the separate class. Somehow in the real project, we always used URIs directly in the classes where we need it without creating a separate class for it. So please use URI in the class you need and remove this class.

Reply:
removed